cradlepoint/util/tableColumns.js

```diff
@@ -39,16 +38,16 @@ const testColumns = [
 const BOMColumns = [
     { field: '_id', headerName: 'ID', headerClassName: 'header', flex: 1},
     { field: 'deviceName', headerName: 'Device Name', headerClassName: 'header', flex: 1},
-    { field: 'optional', headerName: 'Optional', headerClassName: 'header', flex: 1},
+    { field: 'isOptional', headerName: 'Optional', headerClassName: 'header', flex: 1},
     { field: 'quantity', headerName: 'Quantity', headerClassName: 'header', flex: 1},
-    { field: 'physicalOrSoftware', headerName: 'Physical/Software', headerClassName: 'header', flex: 1},
-    { field: 'codeVer', headerName: 'Code Version', headerClassName: 'header', flex: 1},
+    { field: 'deviceType', headerName: 'Device Type', headerClassName: 'header', flex: 1},
+    { field: 'codeVersion', headerName: 'Code Version', headerClassName: 'header', flex: 1},
     { field: 'SKU', headerName: 'SKU', headerClassName: 'header', flex: 1}];
```

Comment:
Again I think I have code that makes this a little bit simpler since it pulls all the needed stuff for the BOM when it gets a test plan